include/bluetooth/gatt.h

```diff
@@ -91,6 +91,13 @@ enum {
 	 * authorized but no data shall be written.
 	 */
 	BT_GATT_WRITE_FLAG_PREPARE = BIT(0),
+
+	/** Attribute write command flag
+	 *
+	 * If set, indicates that write operation is a command which doesn't
```

Comment:
`If set, indicates that the write operation is a command (Write Without Response), which doesn't generate any response`